defaults/main.yaml

```diff
@@ -277,4 +287,4 @@ clamav_freshclam_config__custom: {}
 # Final FreshClam config
 clamav_freshclam_config: "{{
   clamav_freshclam_config__default.update(clamav_freshclam_config__custom) }}{{
-  clamav_freshclam_config__default }}"
+  clamav_freshclam_config__default | dict2items | rejectattr('value', 'sameas', False) | list | items2dict }}"
```

Comment:
Why is this needed?

Reply:
I need your input on this.

If we do this,

```yaml
clamav_scan_command_options__default:
  - --config-file={{ clamav_clamd_file }}
  - --log={{ clamav_scan_log_file }}
  - --fdpass
  - --infected
  - "{{ clamav_client_only | ternary('--stream', None) }}"
```

Then the option becomes an empty value and not "null" (None).

The attribute test can then be rewritten as:

```yaml
clamav_freshclam_config: "{{
  clamav_freshclam_config__default.update(clamav_freshclam_config__custom) }}{{
  clamav_freshclam_config__default | dict2items | rejectattr('value', 'sameas', '') | list | items2dict }}"
```

And /etc/clamd.conf is left with empty values,

```
AllowSupplementaryGroups ""
DetectBrokenExecutables ""
LocalSocket ""
```

CentOS 7 ships with Jinja 2.8, which doesn't have the equalto test, which I believe would fix the condition:

```yaml
{{
  clamav_freshclam_config__default | dict2items | rejectattr('value', 'equalto', '') | list | items2dict }}
```

Is there a way to preserve the None/null setting in a ternary match that can be filtered out on CentOS 7? I don't believe there is, which is why I resorted to False that can be filtered out.